tests/unit/utils/indexData.spec.js

```diff
@@ -0,0 +1,319 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2022 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import {
+  convertChildrenArrayToObject,
+  flattenNestedData,
+  getAllChildren,
+  getChildren,
+  getParents,
+  getSiblings,
+  hashCode,
+} from 'docc-render/utils/indexData';
```

Comment:
I was thinking that maybe we could consider replacing the name of "indexData" with "NavigatorData". What do you think?

Reply:
It makes sense since most of these methods are used in the `NavigatorCard` component

Reply:
Resolved e39a36f46e04ffc43c726b27d5dda242e891dc21